### PR TITLE
environment fixed, setup explained, data process improved, CHANGE: learning rate scheduler

### PR DESCRIPTION
**Pull request by bkmi**
I fixed many problems with installing and running this code in this repository.
1. I explained how to make the `.env` file
2. I created an `environment.yaml` file that actually works
3. I added a config for `alex_mp_20.yml`. See [this link](https://github.com/omri1348/SGFM/blob/main/data/alex_mp_20/README.md) for how to get the data.
4. I created a processing script that makes it much simpler to get the data in `.pkl` format.
5. I removed wandb logging because it was annoying
6. ACTUAL CHANGE: I changed the lr_scheduler to `torch.optim.lr_scheduler.CosineAnnealingLR`, this is merely because doing the validation was slowing things down and getting it to sync and monitor was frustrating.

Hopefully the fork can help people use this code in the future with fewer frustrations!

### CODE REVIEW

**Comment by jiaor17** [comment]
Hi,

Thank you very much for your contributions. I have merged most of the updates. More specifically:

1&2. I kept the environment-related supports. Really appreciate the cleanup!
3. I added an additional link to the repository that provides the alex-mp-20 data split.
4. I kept the new `data/process.py` script.
5. I added a new environment variable `USE_WANDB_LOGGING` in `.env` to control whether wandb is enabled.
6. I kept the original `ReduceLROnPlateau` scheduler as the default, and also included a configuration option with `CosineAnnealingLR` for training efficiency.

Thanks again for helping make this repository easier to use!

**Comment by bkmi** [comment]
Thanks for making DiffCSP++ :)
